src/adaptors/adaptor_common.c

```diff
@@ -32,6 +32,10 @@
 #include <stdatomic.h>
 
 ALLOC_DEFINE(qd_adaptor_config_t);
+const char* LISTENER_OBSERVER_AUTO  = "auto";
```

Comment:
The code convention is to prepend the pointer type to the variable itself, not the data typed definition.  IOW:
```
const char *foo;
```
not:
```
const char* foo;
```
The latter format is incorrect since the pointer attribute is not associated with the type but with the variable. Example:
```
const char* foo, bar;  // multiple vars per line totally legal C
```
Defines a pointer to constant name 'foo' and a constant char bar (not a pointer!)

Which reminds me, I should tell you about my favorite posix function 'atexit' at some point.

Reply:
> Which reminds me, I should tell you about my favorite posix function 'atexit' at some point.

Never heard of that function !!